bcp/client/src/webpack/App.js

```diff
@@ -1,17 +1,17 @@
 import React, {useState} from 'react';
 import {Dropdown} from 'react-bootstrap';
-import Map from './pages/map';
-import List from './pages/list';
+import Map from './pages/Map';
+import List from './pages/List';
 import './less/app.less';
 import {ASSET_MODE} from "./constant";
 import { observer } from "mobx-react";
 import useStores from "./hooks/use-stores";
+import './css/thirdparty/font-awesome/css/font-awesome.css';
```

Comment:
Can we please not bring thirdparty libs unless we have to. Can we not use https://www.npmjs.com/package/@fortawesome/react-fontawesome instead?